### ISSUE TITLE
Rename `codemod` to transform?

### ISSUE THREAD

**Issue by rwjblue** [rwjblue/codemod-cli#23]
If we support template transforms, are they really just "codemods"?  Is a template code?

**Comment by ro0gr** [comment]
Agree. Naming should be aligned. It was surprising to me to use `codemod-cli generate codemod` command to generate contents of `transforms/` directory.

**Comment by rwjblue** [comment]
Thanks for chiming in!

**Comment by rajasegar** [comment]
I assume we are in agreement for the naming here, can we then close this issue?

**Comment by rwjblue** [comment]
If we keep using codemod, we should rename the folder to codemods I guess...
